grunttasks/amdcheck.js

```diff
@@ -12,8 +12,6 @@ module.exports = function (grunt) {
       files: [{
         expand: true,
         src: [
-          'app/{scripts,tests}/**/*.js',
```

Comment:
amdcheck is no longer needed in the `app` directory because eslint throws an error any time there are unused variables.